ietf/status/views.py

```diff
@@ -0,0 +1,46 @@
+# Copyright The IETF Trust 2016-2024, All Rights Reserved
+# -*- coding: utf-8 -*-
+
+import json
+from django.http import HttpResponse, HttpResponseRedirect, HttpResponseNotFound
+from ietf.utils import markdown
+from django.shortcuts import render, get_object_or_404
+from ietf.status.models import Status
+
+import debug                            # pyflakes:ignore
+
+def get_context_data():
+    status = Status.objects.filter(active=True).order_by("-date").first()
+    if status is None:
+        return { "hasMessage": False }
+
+    context = {
+        "hasMessage": True,
+        "id": status.id,
+        "slug": status.slug,
+        "title": status.title,
+        "body": status.body,
+        "url": f"/status/{status.slug}",
```

Comment:
This ought to use the `reverse()` method (that we usually alias to `urlreverse()` in tests) to get the path to the correct view so that we're not baking paths into the code.